lexicons/tools/ozone/set/upsertSet.json

```diff
@@ -0,0 +1,24 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.sets.upsertSet",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Create or update set metadata",
+      "input": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "ref",
+          "ref": "tools.ozone.sets.defs#set"
+        }
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "ref",
+          "ref": "tools.ozone.sets.defs#set"
```

Comment:
Should these set outputs be `#setView` rather than `#set`?  I am sort of assuming `#set` is designed for writing and `#setView` for reading.

Reply:
yep. I was thinking these wouldn't need the value count but I guess it doesn't hurt to unify these all to setView.